application-task-test/application-task-test-docker/src/test/it/com/xwiki/task/test/ui/TaskManagerIT.java

```diff
@@ -247,7 +253,87 @@ void taskReport(WikiReference wiki, TestUtils setup)
 
     @ParameterizedTest
     @WikisSource()
-    @Order(7)
+    @Order(63)
+    void copyPage(WikiReference wiki, TestUtils setup) throws Exception
+    {
+        setup.setCurrentWiki(wiki.getName());
+
+        DocumentReference testRef = new DocumentReference(new LocalDocumentReference(Arrays.asList("Main",
+            "CopyMovePage"), "WebHome"), wiki);
+        setup.createPage(testRef, "{{task reference=\"/Tasks/Task_1\"}}Hello{{/task}}", "CopyMovePage");
+        // Copying a page containing tasks should update their ids and change their owners.
+        setup.gotoPage(testRef);
+        // Gather all task ids for comparison.
+        ViewPageWithTasks viewPage = new ViewPageWithTasks();
+        List<String> macroids = new ArrayList<>();
+        for (int i = 0; i < viewPage.getTaskMacros().size(); i++) {
+            macroids.add(viewPage.getTaskMacroLink(i).getText());
+        }
+        // Copy page.
+        CopyPage copyPage = viewPage.copy();
+        WebElement title = setup.getDriver().findElement(By.id("targetTitle"));
+        title.clear();
+        title.sendKeys("Copy");
+        title.sendKeys(Keys.ENTER);
```

Comment:
Is there any reason why you use this quite complicated method instead of just `copyPage.getDocumentPicker().setTitle("Copy")` and then `clickCopyButton()` which returns the status page?